src/plugins/__tests__/__snapshots__/react-jss.js.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`1. basic 1`] = `
+"
+import React from 'react'
+import injectSheet from 'react-jss'
+
+const styles = {
+  button: {
+    fontWeight: 'bold',
+  },
+}
+
+const Button = ({classes, children}) =>
+  React.createElement('button', {className: classes.button})
+
+export default injectSheet(styles)(Button)
+
+    👇
+
+import React from 'react'
+import injectSheet from 'react-jss'
+
+const styles = {
+  button: \\"button-0-0\\"
```

Comment:
Sounds great! Feel free to make any changes you like. The API is currently in flux so feel free to propose whatever you like to make it easier to plug into so it works with any CSS-in-JS lib :+1: